app/app/fixtures/economy.json

```diff
@@ -24525,5 +24390,24 @@
         "conversion_rate_id": "harmony",
         "conversion_rate_source": "coingecko"
     }
+  },
+  {
+      "model": "economy.token",
+      "pk": 374,
+      "fields": {
+          "created_on": "2022-04-22T12:35:27Z",
+          "modified_on": "2022-04-22T12:36:06.267Z",
+          "address": "0x0000000000000000000000000000000000000000",
+          "symbol": "ETH",
+          "network": "testnetrpc",
+          "decimals": 18,
+          "priority": 1,
+          "chain_id": 1,
+          "network_id": 1,
+          "metadata": {},
+          "approved": true,
+          "conversion_rate_id": null,
+          "conversion_rate_source": null
+      }
```

Comment:
^ does this mean we don't want ETH for mainnet / testnet ?

Reply:
These seemed to be duplicates (correct me if I am wrong).
If you check the file,  you will see that there are 3 entries for ETH: mainnet, rinkeby and testnetrpc (this I use for the tests).

Reply:
Oooo you are right. I wonder why the duplicates were present :O